tests/system/action/poll/test_create.py

```diff
@@ -102,3 +107,318 @@ def test_create_three_options(self) -> None:
         assert option_4.get("used_as_global_option_in_poll_id") == 1
         assert option_4.get("meeting_id") == 112
         assert option_4.get("weight") == 1
+
+
+class CreatePollTestCase(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.create_model(
+            "assignment/1",
+            {
+                "title": "test_assignment_ohneivoh9caiB8Yiungo",
+                "open_posts": 1,
+                "candidate_ids": [1],
+                "meeting_id": 113,
+            },
+        )
+        self.create_model("meeting/113", {})
+        self.create_model("organisation/1", {"enable_electronic_voting": True})
+
+    def test_simple(self) -> None:
```

Comment:
check, that the global option is created

Reply:
Checked in `test_create_correct`